service/crawlingContentService.js

```diff
@@ -0,0 +1,118 @@
+const puppeteer = require("puppeteer");
+
+const getCrawlingContentKeyword = async (req, res) => {
+  const decodedUrl = decodeURIComponent(req.params.url);
+  const keyword = req.query.keyword;
+  const browser = await puppeteer.launch({ headless: true });
+  const TIMEOUT = 20000;
+
+  try {
+    const page = await browser.newPage();
+    await page.goto(decodedUrl);
+
+    const title = await page.$eval("title", (element) => element.textContent);
+    let innerText = await page.$eval("body", (body) => body.innerText);
+
+    const hasTitleKeyword = title.includes(keyword);
+    let hasKeyword = innerText.includes(keyword);
+
+    if (!innerText) {
+      await page.waitForSelector("iframe", { timeout: TIMEOUT });
+
+      const iframeUrl = await page.$eval("iframe", (iframe) => iframe.src);
+      await page.goto(iframeUrl);
+
+      const hasiframeUrlOfNaver = iframeUrl.startsWith(
+        "https://blog.naver.com"
+      );
+      innerText = await page.evaluate(() => document.body.innerText);
+      hasKeyword = innerText.includes(keyword);
+
+      if (!iframeUrl || !hasiframeUrlOfNaver) {
+        throw new Error(`[Invalid iframe URL]`);
+      }
+    }
+
+    const keywordSentence = getAllSentence(innerText).find((sentence) =>
+      sentence.includes(keyword)
+    );
+
+    let urlText = "";
+    if (keywordSentence) {
+      urlText = getKeywordSentence(keywordSentence, keyword);
+    }
+
+    if (hasKeyword) {
+      return res.status(200).json({
+        url: req.params.url,
+        hasTitleKeyword: hasTitleKeyword,
+        urlTitle: title,
+        hasKeyword: hasKeyword,
+        urlText: urlText,
+      });
+    } else {
+      return res.status(200).send({ message: `[This keyword does not exist]` });
+    }
+  } catch (error) {
+    if (isCheckTrueThisUrl(decodedUrl) === false) {
```

Comment:
```suggestion
    if (!isCheckTrueThisUrl(decodedUrl)) {
```
airbnb 컨벤션에 따라 부정연산자를 사용하면 더 좋을 것 같습니다!